src/Android/Xamarin.Android/Samples/Data/EditFeatureLinkedAnnotation/EditFeatureLinkedAnnotation.cs

```diff
@@ -0,0 +1,228 @@
+// Copyright 2021 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.Graphics;
+using Android.OS;
+using Android.Widget;
+using ArcGISRuntime;
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ArcGISRuntimeXamarin.Samples.EditFeatureLinkedAnnotation
+{
+    [Activity(ConfigurationChanges = Android.Content.PM.ConfigChanges.Orientation | Android.Content.PM.ConfigChanges.ScreenSize)]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        name: "Edit features with feature-linked annotation",
+        category: "Data",
+        description: "Edit feature attributes which are linked to annotation through an expression.",
+        instructions: "Pan and zoom the map to see that the text on the map is annotation, not labels. Tap one of the address points to update the house number (AD_ADDRESS) and street name (ST_STR_NAM). Tap one of the dashed parcel polylines and tap another location to change its geometry. NOTE: Selection is only enabled for points and straight (single segment) polylines.",
+        tags: new[] { "annotation", "attributes", "feature-linked annotation", "features", "fields" })]
+    [ArcGISRuntime.Samples.Shared.Attributes.OfflineData("74c0c9fa80f4498c9739cc42531e9948")]
```

Comment:
I think for sample solution generator to work, there needs to be a layout file attribute IIRC